<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>1
abc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:yellow</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frameEnKeyBord">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>200</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color: rgb(216, 240, 250); border-radius: 10px;}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="EightEN">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>155</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 18px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>8
stu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TowEN">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>45</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 18px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>2
abc</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Char1">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>210</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 23px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RemoveAll">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>265</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 23px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Remove">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>265</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 23px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>退格</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FiveEN">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 18px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>5
jkl</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SevenEN">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>155</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 18px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>7
pqr</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ZeroEN">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 18px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>0
yz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ThreeEN">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>45</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 18px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>3
def</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="OneEN">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>45</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 18px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>1
@/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Char2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>210</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 23px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Char3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>265</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 23px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FourEN">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>100</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 18px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>4
ghi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NineEN">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>155</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 18px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>9
vwx</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SixEN">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>100</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{outline: none; border-radius: 5px; font-weight: bold; font-size: 18px; color: rgb(25, 25, 112); background-color:rgb(171, 223, 245); } QPushButton:pressed{background-color: rgb(25, 25, 112) ; color: white;}</string>
     </property>
     <property name="text">
      <string>6
mno</string>
     </property>
    </widget>
    <widget class="QLabel" name="ShowEN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>180</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 23px; color: rgb(25, 25, 112);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
